CMakeLists.txt

```diff
@@ -7,26 +7,149 @@ project(adobe_source_libraries CXX)
 
 set(CMAKE_CXX_EXTENSIONS OFF)
 set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_XCODE_ATTRIBUTE_CLANG_CXX_LANGUAGE_STANDARD "c++0x")
+set(CMAKE_XCODE_ATTRIBUTE_CLANG_CXX_LIBRARY "libc++")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fconstexpr-depth=1024 -ftemplate-depth=1024")
```

Comment:
That's a great question, @ilelann. All I can tell you is Xcode wouldn't build something without the flag added where I put it. I didn't see it in the compiler setting when building one of the test apps (I think it was the FNV one.) I'm not well versed in CMake, so it's possible I've missed something. If I have, let me know.